config/settings.yml

```diff
@@ -1440,13 +1440,7 @@ lighthouse_health_immunization:
     - "patient/Location.read"
 
 mobile_lighthouse_letters:
-  client_id: '0oajpx78t3M8kurld2p7'
```

Comment:
We're working with the letters team and they're adding their own keys which will contain all of this info. We only need client id. The plan is to use our client id when making calls so that the lighthouse team knows who's using the API.